components/DateRangeInput/story.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import moment from "moment"
+import DateRangeInput from "./"
+
+import { text } from "@storybook/addon-knobs"
+
+import 'react-datetime/css/react-datetime.css'
+
+const now = moment()
+
+export default function DateRangeInputStory(stories) {
+  stories.add("DateRangeInput", () => {
+    const props = {
+      fromDate: text("From Date", now),
```

Comment:
Since it's a `DateRangeInput` do you think that naming the props as `from` and `to` would be enough?

```jsx
<DateRangeInput from={...} to={...} />
```